src/components/button-toggle/button-toggle.style.ts

```diff
@@ -176,19 +220,22 @@ StyledButtonToggle.defaultProps = { theme: baseTheme };
 
 export interface StyledButtonToggleWrapperProps {
   grouped?: boolean;
+  variant: Required<ButtonToggleGroupProps["variant"]>;
 }
 
 const StyledButtonToggleWrapper = styled.div<StyledButtonToggleWrapperProps>`
   display: inline-block;
   vertical-align: middle;
 
-  ${({ grouped }) =>
+  ${({ grouped, variant }) =>
     css`
       ${!grouped &&
       css`
         &&&& {
           ${StyledButtonToggle} {
-            border-radius: var(--borderRadius400);
+            border-radius: ${variant === "default"
```

Comment:
Has this been unintentionally reverted? Weren't we checking for minor here before like we are below?

Reply:
It was definitely not intentional - thank you for spotting that -  I will fix it.

Reply:
Nothing else from me, apart from this